content/release/v3/ReleaseNotes326.md

```diff
@@ -0,0 +1,106 @@
+* CONTRAST-7807: Inaccurate number of classes are being reported
+* CONTRAST-7814: Increase .NET Agent Service's max message size
+* CONTRAST-7759: Improve concurrency handling and performance of .NET Instrumentation 
+
+## NodeJS Agent
+* CONTRAST-7222: Tag range commands
+* CONTRAST-7427: Allow config to accept serviceKey for better matching
+* CONTRAST-7234: Propogation
+* CONTRAST-7561: Node: Relative App Directory
+* CONTRAST-7823: Node.JS Agent Download Missing from Log
+
+
+## TeamServer
+* CONTRAST-5498: Duplicated properties in hub.properties and general.properties
+* CONTRAST-7685: Issues sent to JIRA should put HTTP requests/code in code block
+* CONTRAST-7730: Hard Copy Reports Showing Incorrect Start Date in the Executive Summary
+* CONTRAST-7607: Add 'Return to Login' link to Expired License page
+* CONTRAST-7655: Add Play onboarding instructions
+* CONTRAST-4166: TeamServer Notifications
+* CONTRAST-6574: Text on Screen Improvement for AD and LDAP Group Screen
+* CONTRAST-7496: Add Application Wizard
+* CONTRAST-7602: Prevent unlimited TSV backup code attempts
+* CONTRAST-7606: Compress Vulnerability Downloads into Zip Format Prior to Download
+* CONTRAST-6797: utf8 charset support
+* CONTRAST-7015: Update message on delete app modal
+* CONTRAST-7481: Order the App Mappings by Parent Application in Alphabetical Order
+* CONTRAST-7537: Application -> Libraries Grid - Dates overflow
+* CONTRAST-7539: Applications - Applying filter doesn't honor search text
+* CONTRAST-7543: Vulnerabilities Sticky Header Stacks
+* CONTRAST-7608: LDAP Failing to catch exception
+* CONTRAST-7627: All vulnerabilities screen includes vulnerabilities for archived apps
+* CONTRAST-7649: Exporting traces doesn't include properties or evidence
+* CONTRAST-7762: Library Pages Missing Filter by Server
+* CONTRAST-7744: Unable to Load TSV Login Properties
+* CONTRAST-7806: URL's marked as Vulnerable in the Activity tab that are then reported as "No Vulnerability Found" in the Vulnerability tab
+* CONTRAST-7837: Audit log should be pushing INFO level statements, not WARN
+* CONTRAST-7302: Confusion about the username in AD/LDAP
+* CONTRAST-7803: Library Page - Filter by Server
+* CONTRAST-7819: Get Application Endpoint - Does not return the 'notes' field
+* CONTRAST-5107: Header injection findings should specify the header name
+* CONTRAST-7683: TeamServer can't handle Traces w/o <stack/> element
+
+
+## Crawler
+* CONTRAST-6589: Add the ability to pass PhantomJS options in Crawler
+* CONTRAST-5049: Add the ability to specify HTTP headers in crawl requests
+
+*Special Note for Crawler Users* 
+The new release of Crawler and changes to TeamServer will force old crawler configurations to become obsolte. Customers running a previous version of Crawler will not be able to use older versions of their Crawler with TeamServer 3.2.6 or higher. Any customer using the 3.2.6 version of Crawler will have to configure their Crawler to work with TeamServer 3.2.6 or higher. 
```

Comment:
The second sentence can be changed to "Previous versions of Crawler (3.1.7, 3.2.2, or 3.2.3) will not work with TeamServer 3.2.6 or higher." 

And the last sentence isn't necessarily true. The new configuration defaults the previous configurations new values so an "old" configuration can just be executed with no issues.

My 2 cents